bitnami/kube-prometheus/templates/prometheus/config-reloader-service.yaml

```
@@ -0,0 +1,56 @@
+{{- if and .Values.prometheus.enabled .Values.prometheus.configReloader.serviceMonitor.enabled }}
```

I think the service shouldn't depend on the serviceMonitor.enabled value, but in an apart service.enabled value:

```
prometheus:
    configReloader:
        service:
            enabled: true
            serviceMonitor:
                enabled: true
```

```suggestion
{{- if and .Values.prometheus.enabled .Values.prometheus.configReloader.service.enabled }}
```

---

Do we want to have both Service and ServiceMonitor to be enabled by default?

---

I think we should not leave both by default.